jekyll/_cci2/language-python.md

```diff
@@ -102,11 +86,11 @@ jobs:
 #### b. Deploy the app
 {: #deploy-the-app }
 
-In this example, we are choosing to deploy to Heroku. This can be done using the official Heroku orb by adding a new line into our orb section. The Heroku orb contains a set of prepackaged CircleCI configurations you can use to deploy applications to Heroku. Learn more about the [Heroku orb]({{site.devhub_base_url}}/orbs/orb/circleci/heroku).
+In this quickstart, we are going to deploy to Heroku. We can do this using the official Heroku orb by adding a new line into our orb section. The Heroku orb contains a set of prepackaged CircleCI configurations you can use to deploy applications to Heroku. Learn more about the [Heroku orb]({{site.devhub_base_url}}/orbs/orb/circleci/heroku).
```

Comment:
Maybe link out to Heroku's main page on the first mention of them? Also maybe "quickstart guide" or "guide" over "quickstart"?